pydid/service.py

```diff
@@ -38,5 +37,27 @@ class Config:
     priority: int = 0
 
 
+DIDCommService = DIDCommV1Service
+
+
+class DIDCommV2ServiceEndpoint(ServiceEndpoint):
```

Comment:
Piggybacking on this PR because I was looking at adding a similar thing.

Since the [W3C spec](https://www.w3.org/TR/did-spec-registries/#didcommmessaging) supports AIPs with the `accept` field, I think it is correct that `DIDCommMessaging` and the `accept` are valid to use for didcomm v1 services. Right ?

Maybe splitting the `DIDCommService` structures between the "aries flavor" of `did-communication` vs the W3C spec is a more sensible way to go than v1 vs v2 ?